.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,18 @@
+name: "unit test"
+
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node: [4, 6, 8, 10, 12, 14]
```

Comment:
I included 10, 12 and 14.

Worth noting the output on 14 includes lots of warnings of the form: `Warning: Accessing non-existent property 'cd' of module exports inside circular dependency`. I believe this because node 14 doesn't allow circular imports anymore.

It's possibly caused by a dev dependency that needs updating but not clear which.

Reply:
> It's possibly caused by a dev dependency that needs updating but not clear which.

Looks like it's eslint or one of their dependencies since it only happens in the lint step

I think it's fine to leave it as-is or even remove eslint entirely — we don't really change this repo so I don't think linting is important